src/plugin.js

```diff
@@ -1,3 +1,4 @@
+/* global DocumentFragment */
```

Comment:
This is fine for the PR. I'll make a note to fix it properly, (unless you want to give it a shot https://github.com/videojs/videojs-contrib-hls/pull/788)